cloudinfo/containers.go

```diff
@@ -47,3 +49,68 @@ func (infoSvc *CloudInfoService) GetClusterConfigPath(clusterId string, basePath
 
 	return configDetails.FilePath, nil
 }
+
+// GetAlbState retrieves the details of State corresponding to ALB ID
+// albId: the ID of the ALB
+// Returns the State for an ALB ID.
+func (infoSvc *CloudInfoService) GetAlbState(albId string) (status string, err error) {
```

Comment:
Return values usually do not have a variable name, just a data type. For instance your return configuration here should just be "(string, error)" and drop the "status" and "err".

We have been leaving variable names out of the return, and we set them up as locals inside the function instead.

This applies to all your functions.

Reply:
Done, removed the named return values.